docs/quickstart.rst

```diff
@@ -29,7 +29,7 @@ Getting Started with Docker Compose
 
 Docker Compose is a powerful tool that enables you to launch multiple Docker images in a coordinated fashion.  It is ideal for platforms like Confluent.  Before you get started, you will need to install both the core `Docker Engine <https://docs.docker.com/engine/installation/>`_ and `Docker Compose <https://docs.docker.com/compose/install/>`_.  Once you've done that, you can follow the steps below to start up the Confluent Platform services.
 
-#. Create and configure the Docker Machine.
+#. Create and configure the Docker Machine (Windows Only)
```

Comment:
i'd add this at the beginning: `**Windows Only:** Create and configure the Docker Machine.`